src/v/cloud_io/remote.h

```diff
@@ -304,7 +321,69 @@ class remote final
 
     const io_resources& resources() const { return *_resources; }
 
+    /// Event filter class.
+    ///
+    /// The filter can be used to subscribe to subset of events.
```

Comment:
Update the comment to match cloud_io vocabulary.

Reply:
yes please. keeping all this updated is really helpful for AI tools too.